tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -159,3 +166,60 @@ Verify image manifest digest against vanilla docker
     Log  ${output}
     Should Be Equal As Integers  ${rc}  0
     Should Contain  ${output}  sha256:be3c11fdba7cfe299214e46edc642e09514dbb9bbefcd0d3836c05a1e0cd0642
+
+Attempt docker pull mitm
+    Wait Until Keyword Succeeds  5x  15 seconds  Pull image  gigawhitlocks/docker-layer-injection-proxy:latest
```

Comment:
Consider the hit by a bus rule: worth putting this image in vmware's bintray or dockerhub account?

Reply:
I was thinking of adding the folder I used to build them, with the 2 dockerfiles, to the repo? I wasn't sure and was hoping @mhagen-vmware could give me some feedback as to the best approach for that. The registry one, in particular, is a fork of this repo https://github.com/docker/distribution-library-image but they don't have a license on the repo? So I'm not sure if I can check that in?

Reply:
I guess yeah I could just put the images somewhere, although if I get hit by a bus my dockerhub will live on. You'd just have to pull them down in the case dockerhub was being EOL'd.

Reply:
I really don't like this. we have it already with some of our images though. it should go under vmware IMO. @mhagen-vmware

Reply:
We have a docker hub account already:
https://hub.docker.com/r/victest/

The password is in our secrets

Reply:
and yea, the dockerfiles should be added to the repo

Reply:
Until I know what to do with it for sure, I posted the additional code related to this test (what's running in those two images) here https://github.com/gigawhitlocks/mitmproxy-docker-registry

Reply:
kk

Reply:
also this "we have it already with some of our images though" is the only reason I did it this way initially, lol

Reply:
OK I committed the necessary bits to build those two images. Third party includes are `mitmdump` (MIT License https://github.com/mitmproxy/mitmproxy/blob/master/LICENSE) and the docker registry image Dockerfile (edited) which is Apache licensed https://github.com/docker/distribution/blob/master/LICENSE

Reply:
We don't compile against those so I don't exactly know if the LICENSE matters, but Apache & MIT should be OK regardless.